skeleton/.buildkite/pipeline.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - label: Check Hydra evaluation of release.nix
+    commands:
+      - "cd skeleton"  # TODO: remove this line
+      - "nix-build -A iohkLib.check-hydra -o check-hydra.sh"
+      - "./check-hydra.sh"
+    agents:
+      system: x86_64-linux
+
+  # This will ensure that the generated Nix code is kept up to date.
+  - label: Check auto-generated Nix
+    commands:
+      - "export NIX_PATH=iohk_nix=$PWD:$NIX_PATH"  # TODO: remove this line
```

Comment:
Ohhhhhhh I think I see now, we need these `cd` since the project is not in the root of the repo.